tools/benchmarks/tools.cc

```diff
@@ -0,0 +1,36 @@
+#include "tools.h"
+#include <opencv2/opencv.hpp>
+#include <dirent.h>
+
+using namespace std;
```

Comment:
I don't know if you want to keep those two lines since you have them in your .h that you include.